### DISCUSSION TITLE
Radix Dropdown Trigger inside Swiper not triggering with wrapped text

### DISCUSSION THREAD

**Discussion by abradle2** [radix-ui/primitives#2740] [Help]
I'm looking for help understanding a phenomenon I'm seeing, involving the interplay between radix and swiperjs. I suspect there's something around focusable elements involved, but I'm currently at a loss. Please tell me to ask the Swiper community if it makes more sense - asking here because I'm keen to learn more about the underlying mechanisms.

[Code Sandbox](https://codesandbox.io/p/sandbox/festive-frog-88hpr3)

**Problem**
I have a basic `Swiper` with a `SwiperSlide`. Inside, there is a simple `DropdownMenu` with a `button` as a trigger. If the `button` contains plain, unwrapped text (e.g. `<button>click here</button>`), then clicking it works as expected and the `DropdownMenu.Contents` is added to the DOM. If the `button` contains text wrapped in a `span` (`<button><span>click here</span></button>`), text wrapped in a `div`, or even an `svg`, then clicking it does not trigger the dropdown to open.

The Code Sandbox link contains a `button` with both wrapped and unwrapped text in it. Click each line to see the different behavior.

**Additional observations**
If you add an `onClick()` handler to the `button` (e.g. `onClick={() => console.log("clicked")}`, then this handler is fired when you click the _wrapped_ text, but not when you click the _unwrapped_ text.

In the `Swiper`, you can add the prop `simulateTouch={false}` ([API doc here](https://swiperjs.com/swiper-api#param-simulateTouch)), and that makes this whole thing magically work as expected - wrapped and unwrapped text both trigger the expected action.

**Comment by benoitgrelard** [comment]
Hey, I am not familiar with `Swiper` but it certainly looks like they are doing some weird stuff with click/pointer events.
As you point out, even with Radix out of the equation (just a plain button in your `Swiper` you get some weird behaviour with the console log already. So yes I think the question is better asked in that community.
